<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>cuteUSB (formerly viusb)</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                    <widget class="QTreeWidget" name="usbTreeWidget">
                        <property name="headerLabels">
                            <stringlist>
                                <string>Bus</string>
                                <string>Device</string>
                                <string>Description</string>
                                <string>Vendor ID</string>
                                <string>Product ID</string>
                            </stringlist>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuFile">
                <property name="title">
                    <string>File</string>
                </property>
                <addaction name="actionExit"/>
            </widget>
            <widget class="QMenu" name="menuHelp">
                <property name="title">
                    <string>Help</string>
                </property>
                <addaction name="actionAbout"/>
                <addaction name="actionAboutQt"/>
            </widget>
            <addaction name="menuFile"/>
            <addaction name="menuHelp"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionExit">
            <property name="text">
                <string>Exit</string>
            </property>
        </action>
        <action name="actionAbout">
            <property name="text">
                <string>About</string>
            </property>
        </action>
        <action name="actionAboutQt">
            <property name="text">
                <string>About Qt</string>
            </property>
        </action>
    </widget>
    <resources/>
    <connections/>
</ui>
